src/controllers/AuthController.js

```diff
@@ -31,6 +34,53 @@ const forgotPassword = async (request, response) => {
   }
 };
 
-export default {
-  forgotPassword
+/**
+ * Update user verified status
+ *
+ * @param {object} req
+ * @param {object} res
+ * @returns {json} - json
+ */
+
+const updateStatus = async (req, res) => {
+  const user = await User.findOne({ where: { id: req.params.id } });
+  const secret = process.env.ACCOUNT_VERIFICATION_SECRET;
+  const token = await jwt.verify(req.params.token, secret, (err, decoded) => {
+    if (err) {
+      return err;
+    }
+    return decoded;
+  });
+
+  if (!user) {
+    return errorResponse(res, 401, 'Sorry could not verify email');
+  }
+  const { verifiedToken } = user;
+
+  if (token.name) {
+    if (token.name === 'TokenExpiredError') {
```

Comment:
instead of this nested `if` statements, how about we merge the two statements into one? `if (token.name && token.name === 'TokenExpiredError')`

Reply:
done